[DOC] fix typo in dataprocess_types.go

**Provide a link to that doc page:**
https://github.com/fluid-cloudnative/fluid/blob/master/api/v1alpha1/dataprocess_types.go
**What is the defect and your suggestions on improvement:**
api\v1alpha1\dataprocess_types.go:99:5: corrected "requried" to "required"

--- a/api/v1alpha1/dataprocess_types.go
+++ b/api/v1alpha1/dataprocess_types.go
@@ -96,7 +96,7 @@ type ScriptProcessor struct {
 // DataProcessSpec defines the desired state of DataProcess
 type DataProcessSpec struct {
 	// Dataset specifies the target dataset and its mount path.
-	// +requried
+	// +required
 	Dataset TargetDatasetWithMountPath `json:"dataset"`
 
 	// Processor specify how to process data.

